dogowner/views.py

```diff
@@ -1,3 +1,10 @@
-# from django.shortcuts import render
+from django.shortcuts import render
+from daycare.models import DayCare
 
-# Create your views here.
+
+def dog_owner_home(request):
+    context = {
+        'dogowner': request.user,
+        'daycares': DayCare.objects.all(),
+    }
+    return render(request, 'dogowner/dog_owner_homepage.html', context)
```

Comment:
Im not sure we need to send 'dogowner' containg User in context because it already include in the request .
In addition , we should use @login_requierd decorater for case when User try to reach this page from URL .

Reply:
Done.